bin/get-amy.py

```diff
@@ -82,4 +89,17 @@ def sort_flags(data):
 
 
 if __name__ == '__main__':
-    main()
+    parser = argparse.ArgumentParser(description=sys.modules[__name__].__doc__)
```

Comment:
Um... it's in the unified-styles branch in my copy of the repo. Please 
add it to 'Makefile' for now and I'll figure it out in the merge - thanks.